src/lib/apis/zodSchemas/avwx/metarSchemas.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+
+export const MetarTimeSchema = z.object({ dt: z.string().datetime() });
+
+export const MetarWindDirectionSchema = z.object({ repr: z.string() });
+
+export const MetarWindSpeedSchema = z.object({ repr: z.string() });
+
+export const MetarVisibilitySchema = z.object({ repr: z.string() });
+
+export const MetarTemperatureSchema = z.object({ repr: z.string() });
+
+export const MetarDewpointSchema = z.object({ repr: z.string() });
+
+export const MetarAltimeterSchema = z.object({ value: z.number() });
+
+export const MetarUnitsSchema = z.object({
+    accumulation: z.string(),
+    altimeter: z.string(),
+    altitude: z.string(),
+    temperature: z.string(),
+    visibility: z.string(),
+    wind_speed: z.string(),
+});
```

Comment:
```suggestion
export const MetarTimeSchema = z.object({ dt: z.string().datetime() });
export const MetarWindDirectionSchema = z.object({ repr: z.string() });
export const MetarWindSpeedSchema = z.object({ repr: z.string() });
export const MetarVisibilitySchema = z.object({ repr: z.string() });
export const MetarTemperatureSchema = z.object({ repr: z.string() });
export const MetarDewpointSchema = z.object({ repr: z.string() });
export const MetarAltimeterSchema = z.object({ value: z.number() });
export const MetarUnitsSchema = z.object({
    accumulation: z.string(),
    altimeter: z.string(),
    altitude: z.string(),
    temperature: z.string(),
    visibility: z.string(),
    wind_speed: z.string(),
});
```
Any reason for the empty lines? Unless some linting requires this, I'd keep it compressed given they are one-liners? (I get it for others where they are more complex.